keps/77-dynamically-sized-jobs/kep.yaml

```diff
@@ -0,0 +1,24 @@
+title: 
+kep-number: 77
+authors:
+  - "@vicentefb"
+status: provisional
+creation-date: 2024-03-14
+reviewers:
+  - "@andrewsykim"
+  - "@alculquicondor"
+  - "@astefanutti"
+approvers:
+  - "@alculquicondor"
+  - "@andrewsykim"
+  - "@astefanutti"
```

Comment:
They are not approver.

Reply:
+1 please remove myself and astefanutti and add us as reviewers only

Reply:
👍🏼

Reply:
Done